main.ts

```diff
@@ -445,27 +484,6 @@ export default class VimrcPlugin extends Plugin {
 
 	}
 
-	captureYankBuffer() {
-		if (this.yankToSystemClipboard) {
-			let currentBuffer = this.codeMirrorVimObject.getRegisterController().getRegister('yank').keyBuffer;
-			if (currentBuffer != this.lastYankBuffer) {
-				if (this.lastYankBuffer.length > 0 && currentBuffer.length > 0 && currentBuffer[0]) {
-					navigator.clipboard.writeText(currentBuffer[0]);
-					navigator.clipboard.readText().then((value) => { this.lastSystemClipboard = value; });
-				}
-				this.lastYankBuffer = currentBuffer;
-				return;
-			}
-			let currentClipboard = navigator.clipboard.readText().then((value) => {

```

Comment:
- [ ] figure out how to make pasting involve system clipboard

Reply:
https://github.com/replit/codemirror-vim/blob/e7e8b1f74d7be0464de0ff45e1980139ffec2b35/src/vim.js#L1162

Reply:
https://github.com/replit/codemirror-vim/blob/e7e8b1f74d7be0464de0ff45e1980139ffec2b35/src/vim.js#L2637

Maybe check whether the argument is the unnamed register (instead of nothing), but getRegister is not only called for pasting.